source/guide_prosody.rst

```diff
@@ -0,0 +1,404 @@
+
+.. include:: includes/open-port.rst
+
+Configure DNS records
+---------------------
+
+Your DNS needs to be setup with the following values:
+
++-------------------------------------------------+-------+-------+-------+----------+--------+----------+-------------------------+
+|                       name                      |  ttl  | class |  type | priority | weight |   port   | target                  |
++-------------------------------------------------+-------+-------+-------+----------+--------+----------+-------------------------+
+|                isabell.uber.space               |  3600 |   IN  |   A   |          |        |          |      35.195.215.42      |
++-------------------------------------------------+-------+-------+-------+----------+--------+----------+-------------------------+
+|                isabell.uber.space               |  3600 |   IN  |  AAAA |          |        |          |            0:           |
++-------------------------------------------------+-------+-------+-------+----------+--------+----------+-------------------------+
+|            upload.isabell.uber.space            |  3600 |   IN  | CNAME |          |        |          | isabell.uber.space      |
++-------------------------------------------------+-------+-------+-------+----------+--------+----------+-------------------------+
+|             xmpp.isabell.uber.space             |  3600 |   IN  | CNAME |          |        |          | isabell.uber.space      |
++-------------------------------------------------+-------+-------+-------+----------+--------+----------+-------------------------+
+| _xmpp-client._tcp.isabell.uber.space            | 18000 | IN    | SRV   | 0        | 5      | C2S-PORT | xmpp.isabell.uber.space |
++-------------------------------------------------+-------+-------+-------+----------+--------+----------+-------------------------+
+| _xmpp-server._tcp.isabell.uber.space            | 18000 | IN    | SRV   | 0        | 5      | S2S-PORT | xmpp.isabell.uber.space |
++-------------------------------------------------+-------+-------+-------+----------+--------+----------+-------------------------+
+| _xmpp-server._tcp.conference.isabell.uber.space | 18000 | IN    | SRV   | 0        | 5      | S2S-PORT | xmpp.isabell.uber.space |
++-------------------------------------------------+-------+-------+-------+----------+--------+----------+-------------------------+
+
+Configure luarocks
+------------------
+
+Prosody is written in ``lua`` and has some runtime dependencies which we install with the package manager ``luarocks``. In order to run the installed packages we have to adapt the path-variable in ``~/.bash_profile``:
+
+::
+
+ PATH=$HOME/.luarocks/bin:$PATH
+ 
+ export PATH
+
+Additionally we need to provide the paths ``LUA_PATH`` as well as ``LUA_CPATH`` and have to reload to use ``luarocks`` accordingly:
+
+::
+
+ [isabell@stardust ~]$ echo "$(luarocks path)" >> ~/.bash_profile
```

Comment:
How about something like this, in case the paths change?
```suggestion
 [isabell@stardust ~]$ echo 'eval "$(luarocks path)"' >> ~/.bash_profile
```